compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -1225,6 +1268,36 @@ class Allocator {
     }
   }
 
+  NOINLINE void *reTagBlock(const Options &Options, void *TaggedPtr,
+                            void *HeaderTaggedPtr,
+                            Chunk::UnpackedHeader *Header, const uptr Size,
+                            bool BypassQuarantine) {
+    DCHECK(useMemoryTagging<AllocatorConfig>(Options));
+    void *Ptr = HeaderTaggedPtr;
```

Comment:
Get rid of this line and just call the argument `Ptr`.

Reply:
Done